### PR TITLE
Testing/loss

### PR DESCRIPTION
**Pull request by DeclanMcIntosh**
updated and working with smoothness, SSIM, and L1 photometric loses

### CODE REVIEW

File: traning.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+from lossFunctions import monoDepthV2Loss
+from dataGen import depthDataGenerator
+
+# define these
+batchSize = 12
+trainingRunDate = '2020_4_18'
+Notes = 'Full_data_no_mu_with_SSIM_on_left_right_only_full_loss_'
+
+# build data generators
+train_generator = depthDataGenerator('../train/left/','../train/right/',batch_size=batchSize, shuffle=True, max_img_time_diff=700 )
```
Comment (robertklee) [inline traning.py]: Let's move some of these constants to `config.py`

Comment (robertklee) [reply traning.py]: easier to modify later on

File: modelDef.py [hunk 2]
```diff
@@ -144,80 +144,75 @@ def resentLayerLaterState_18(inputLayer, inputChannels, channels, poolingStride,
...
     x = concatenate([x,scale_1],axis=3)
 
-    x = Conv2D(256, kernel_size=3, strides=1, data_format='channels_last', padding='same', name="DecoderCov_Block_2_1")(x)
-    x = Conv2D(256, kernel_size=3, strides=1, data_format='channels_last', padding='same', name="DecoderCov_Block_2_2")(x)
-    x = Conv2D(256, kernel_size=3, strides=1, data_format='channels_last', padding='same', name="DecoderCov_Block_2_3")(x)
-    x = UpSampling2D(data_format='channels_last', name="UpSample2")(x)
+    x = Conv2D(512, kernel_size=3, strides=1, data_format='channels_last', padding='same', name="DecoderCov_Block_2_1", activation='relu')(x)
+    x = Conv2D(512, kernel_size=3, strides=1, data_format='channels_last', padding='same', name="DecoderCov_Block_2_2", activation='relu')(x)
+
+    x = Conv2DTranspose(256, (2, 2), strides=(2, 2), data_format='channels_last', name="ConvTranspose2")(x)
```
Comment (robertklee) [inline modelDef.py:164]: any reason we changed from upsampling2D to Conv2DTranspose?

Comment (DeclanMcIntosh) [reply modelDef.py:164]: to properly reflect unet they mention for decoder

**Pull request merged**